blockdag/sync_rate.go

```diff
@@ -0,0 +1,52 @@
+package blockdag
+
+import "time"
+
+const syncRateWindowDuration = 15 * time.Minute
+
+// addBlockProcessTime adds the last the last
```

Comment:
typo the last the last.

Also I'm not sure why break the comment into such short lines.